app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,15 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    def __init__(self, favourite_dish: str) -> None:
+        self._favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self) -> None:
+        print(f"{self.nickname} is eating {self._favourite_dish}")
+
+    def get_rating(self) -> str:
```

Comment:
it is not what you were supposed to do. instead of implementing this methods, just inherid from ABC
`
class Dwarf(Player, ABC):`